src/main/webapp/js/finscholar/scholarshippageview/scholarshippageview.soy

```diff
@@ -0,0 +1,109 @@
+{namespace example.templates.scholarshippageviews}
+{template .scholarshippage}
+  {@param scholarship: [generalInfo: [scholarshipName: string,  
+                                       schoolsList:string,
+                                       introduction: string,
+                                       URL: uri],
+                        requirements: [academicRequirements: string,
+                                       ethnicityRaceRequirements: string,
+                                       financialRequirements: string,
+                                       genderRequirements: string,
+                                       locationRequirements: string,
+                                       nationalOriginRequirements: string,
+                                       otherRequirements: string],
+                        applicationNotes: [isRenewable: bool,
+                                           amountPerYear: string,
+                                           applicationProcess: string,
+                                           numberOfYears: int]
+                        ]}
+  <section class="section--center mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
+    <div class="mdl-card__supporting-text mdl-grid mdl-grid--no-spacing">
+      <div class="page-content">
+        {call .scholarshipIntroduction}
+          {param generalInfo: $scholarship.generalInfo /}
+        {/call}
+        {call .scholarshipRequirements}
+          {param requirements: $scholarship.requirements /}
+        {/call}
+        {call .scholarshipApplication}
+          {param applicationNotes: $scholarship.applicationNotes /}
+        {/call}
+      </div>
+    </div>
+  </section>
+{/template}
```

Comment:
I split the main template for scholarship into three small templates. Not sure if this is appropriate. For now it seems a little redundant but later I'll add more jsactions and styles into each component. By then maybe this structure looks better?

Reply:
What if you put these in separate files? Not sure if it would help with how it's organized.

Reply:
What if you put these in separate files? Not sure if it would help with how it's organized.

Reply:
Emmm semantically all these panel serves the same purpose so I prefer to have them in one file.